src/components/AccountManager/AccountManager.js

```diff
@@ -0,0 +1,253 @@
+import React, {useContext, useState, useEffect} from "react";
+import {Button, Collapse, Input, InputGroup, InputGroupAddon,
+    Alert} from "reactstrap"
+import {AppContext} from "../../context/AppContext";
+import GoogleCalendarConfig from "../Google/GoogleCalendarConfig";
+import FaceLoginSetup from "../Config/FaceLoginSetup/FaceLoginSetup";
+import {useDatabase} from "../../hooks/useDatabase";
+
+const AccountManager = (props) => {
+
+    // First Name State
+    const [firstName, setFirstName] = useState("");
+    const [changeFirstName, setChangeFirstName] = useState("");
+
+    // Last Name State
+    const [lastName, setLastName] = useState("");
+    const [changeLastName, setChangeLastName] = useState("");
+
+    // Email State
+    const [email, setEmail] = useState("");
+    const [changeEmail, setChangeEmail] = useState("");
+
+    // Guest Account?
+    const [isGuest, setIsGuest] = useState(true);
+    const [confirmationText, setConfirmationText] = useState("Confirm");
+
+    //TEMP - PASSWORD STATE
+    const [password, setPassword] = useState(""); //DEV USE ONLY - DELETE AFTER TESTING
+
+    // Reset Password States
+    const [resetPasswordOne, setResetPasswordOne] = useState("");
+    const [resetPasswordTwo, setResetPasswordTwo] = useState("");
+    const [passwordsMatch, setPasswordsMatch] = useState("Enter your new password...");
+
+    // Redraw Page State
+    const [updateInfo, setUpdateInfo] = useState(false);
+
+    // Collapsing Drawer States
+    const [isInfoSetupOpen, setInfoSetupOpen] = useState(false);
+    const toggleInfoSetup = () => {
+        if(isResetPasswordOpen){
+            toggleResetPassword();
+        }
+        setInfoSetupOpen(!isInfoSetupOpen);
+    };
+
+    const [isResetPasswordOpen, setResetPasswordOpen] = useState(false);
+    const toggleResetPassword = () => {
+        if(isInfoSetupOpen){
+            toggleInfoSetup()
+        }
+        setResetPasswordOpen(!isResetPasswordOpen);
+    };
+
+    const [isFaceSetupOpen, setFaceSetupOpen] = useState(false);
+    const toggleFaceSetup = () => setFaceSetupOpen(!isFaceSetupOpen);
+
+    const [isGoogleOpen, setIsGoogleOpen] = useState(false);
+    const toggleGoogle = () => setIsGoogleOpen(!isGoogleOpen);
+
+    const context = useContext(AppContext);
+    const userInfo = useDatabase();
+
+    useEffect(() => {
+        const getUserInfo = async () => {
+            let info = null;
```

Comment:
Good job Nick! Just one advice, so you're declaring "info" and then you make async call to get user info and then you check "info" in "if" statement. No guarantee that gets user info back at the time you check it inside "if" statement.  You could move your "if" statement inside .then()